lib/gracenote/request/search.rb

```diff
@@ -0,0 +1,29 @@
+require "gracenote/request/option"
+
+module Gracenote
+  module Request
+    module Search
+      # @params mode [String] 'SINGLE_BEST' or 'SINGLE_BEST_COVER'
+      # @params range [Hash] { start: 1, end: 10 }
+      #
+      def search(artist: nil, album: nil, track: nil, mode: nil, range: nil)
+        query("album_search", auth: auth, lang: lang, country: country) do |xml|
+          xml.MODE { xml.text mode } if mode
+          if range
+            xml.RANGE {
```

Comment:
__[rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)